src/Jenssegers/Mongodb/Query/Builder.php

```diff
@@ -325,9 +327,7 @@ public function getFresh($columns = [])
             if ($this->limit) {
                 $pipeline[] = ['$limit' => $this->limit];
             }
-            if ($this->projections) {
-                $pipeline[] = ['$project' => $this->projections];
-            }
+
```

Comment:
Hello,

Could you please remove this line so styleci check passes?

Also, please kindly update with the base branch again, so many updates coming to this library finally!

Thank you for your contribution!